### PR TITLE
Added the option to show Audio player speeds upfront

### PR DESCRIPTION
**Pull request by rishabhPlivo**
Here I've added the option to not open the audio player speeds in a popup. That is to have them upfront. Here's a screenshot attached of the same.

Screenshot 2023-08-29 at 5 46 55 PM

### CODE REVIEW

File: src/components/AudioPlayer/components/PlaybackSpeed.js [hunk 1]
```diff
@@ -16,27 +16,17 @@ export default function PlaybackSpeed({ waveSurfer }) {
...
-      <Button className="contacto-player-speed-trigger" type="secondary">
-        {speed}x
-      </Button>
-    </Popover>
+    <div className="contacto-player-speed">
+      {speeds.map((speed) => (
+        <Button
+          key={speed}
+          className="contacto-player-speed-buttons"
+          type={speed === waveSurfer?.getPlaybackRate() ? 'secondary' : 'table-action-link'}
```
Comment (Abhishek-plivo) [inline src/components/AudioPlayer/components/PlaybackSpeed.js:24]: move this to a function

File: src/components/AudioPlayer/components/PlaybackSpeedPopup.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    setSpeed(waveSurfer?.getPlaybackRate())
+  }, [waveSurfer])
+
+  return (
+    <Popover
+      overlayClassName="contacto-player-speed-popup"
+      title="Playback Speed"
+      content={() => (
+        <div>
+          {speeds.map((speed) => (
```
Comment (Abhishek-plivo) [inline src/components/AudioPlayer/components/PlaybackSpeedPopup.js:24]: this chunk can be reused

File: src/components/AudioPlayer/components/PlaybackSpeedPopup.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from 'react'
+import { Popover } from 'antd'
+import { Button } from '../../Button/index'
+
+const speeds = [0.8, 1, 1.2, 1.5, 1.7, 2, 2.5]
+export default function PlaybackSpeed({ waveSurfer }) {
+  const [speed, setSpeed] = useState(waveSurfer?.getPlaybackRate())
+
+  const handleSpeedChange = (speed) => {
```
Comment (Abhishek-plivo) [inline src/components/AudioPlayer/components/PlaybackSpeedPopup.js:9]: handleSpeedChange can be reused from src/components/AudioPlayer/components/PlaybackSpeed.js

File: src/components/AudioPlayer/styles.scss [hunk 4]
```diff
@@ -17,7 +17,7 @@
   }
 
   &.contacto-button--table-action-link {
-    background-color: var(--background-highlight-color);
+    background-color: var(--background-highlight-color) !important;
```
Comment (Abhishek-plivo) [inline src/components/AudioPlayer/styles.scss:20]: is there no other way to apply this style instead of !important

**Review by Abhishek-plivo — DISMISSED** [review]

File: src/components/AudioPlayer/components/PlaybackSpeed.js (hunk 1 above)
Comment (rishabhPlivo) [reply src/components/AudioPlayer/components/PlaybackSpeed.js:24]: moved it

File: src/components/AudioPlayer/components/PlaybackSpeedPopup.js (hunk 2 above)
Comment (rishabhPlivo) [reply src/components/AudioPlayer/components/PlaybackSpeedPopup.js:24]: hanged it into a util method

File: src/components/AudioPlayer/components/PlaybackSpeedPopup.js (hunk 3 above)
Comment (rishabhPlivo) [reply src/components/AudioPlayer/components/PlaybackSpeedPopup.js:9]: moved it to utils and reusing from there

File: src/components/AudioPlayer/styles.scss (hunk 4 above)
Comment (rishabhPlivo) [reply src/components/AudioPlayer/styles.scss:20]: The style was already there but it was getting over-written so added this

File: src/components/AudioPlayer/components/PlaybackSpeedPopup.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    <Popover
+      overlayClassName="contacto-player-speed-popup"
+      title="Playback Speed"
+      content={() => (
+        <RenderButtons
+          speeds={speeds}
+          waveSurfer={waveSurfer}
+          setSpeed={setSpeed}
+          parentClassName="contacto-player-speed-popup"
+          buttonClassName="contacto-player-speed-popup"
```
Comment (samuellawerentz) [inline src/components/AudioPlayer/components/PlaybackSpeedPopup.js:24]: Why the same classname?

**Pull request merged**